### PR TITLE
New package: EchoLink.EchoLink version 2.3.121.1

### LINKED ISSUE #112087: [Package Request]: EchoLink
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.echolink.org/download.htm
Publisher: EchoLink.org
Package Name: EchoLink
Description: EchoLink® software allows licensed Amateur Radio stations to communicate with one another over the Internet, using streaming-audio technology.
Package Version: 2.3.121
Installer URL: https://d34yfhzor7jtt3.cloudfront.net/EchoLinkSetup-2-3-121.exe

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- Resolve #112087

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/113447&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/e/EchoLink/EchoLink/2.3.121.1/EchoLink.EchoLink.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: EchoLink.EchoLink
+PackageVersion: 2.3.121.1
+PackageLocale: en-US
+Publisher: EchoLink.org
```
Comment (Trenly) [inline manifests/e/EchoLink/EchoLink/2.3.121.1/EchoLink.EchoLink.locale.en-US.yaml:7]:
```suggestion
Publisher: Synergenics, LLC
```
This should match control panel

File: manifests/e/EchoLink/EchoLink/2.3.121.1/EchoLink.EchoLink.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: EchoLink.EchoLink
+PackageVersion: 2.3.121.1
+Installers:
+- Architecture: x86
+  InstallerType: burn
+  InstallerUrl: https://d34yfhzor7jtt3.cloudfront.net/EchoLinkSetup-2-3-121.exe
+  InstallerSha256: 29A5E229EACDE0495E4C977F9BB900794BDEAE907079C227BD16DEFF011314F0
```
Comment (Trenly) [inline manifests/e/EchoLink/EchoLink/2.3.121.1/EchoLink.EchoLink.installer.yaml:10]:
```suggestion
- Architecture: x86
  InstallerType: burn
  InstallerUrl: https://d34yfhzor7jtt3.cloudfront.net/EchoLinkSetup-2-3-121.exe
  InstallerSha256: 29A5E229EACDE0495E4C977F9BB900794BDEAE907079C227BD16DEFF011314F0
  ProductCode: '{5ae42bc2-b60c-4a25-b0aa-c25e36aad094}'
```
Two ARP entries are written, so product code is needed to disambiguate

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**